### PR TITLE
Add test to check recorded IP assignments for switch networks

### PR DESCRIPTION
**Pull request by milan-zededa**
Signed-off-by: Milan Lenco <milan@zededa.com>

### CODE REVIEW

File: tests/eclient/testdata/app_dhcp.txt [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+#!/bin/sh
+
+EXP_MAC=$1
+EXP_IP=$2
+EDEN={{EdenConfig "eden.root"}}/{{EdenConfig "eden.bin-dist"}}/{{EdenConfig "eden.eden-bin"}}
+
+function get_ip_assignments {
+    assignments=$($EDEN info InfoContent.niinfo.displayname:switch --tail 1 --out InfoContent.niinfo.ipAssignments)
+    echo "Current IP assignments: $assignments"
+    if echo "$assignments" | grep -q "macAddress:\"$EXP_MAC\"  ipAddress:\"$EXP_IP\""; then
```
Comment (giggsoff) [inline tests/eclient/testdata/app_dhcp.txt:143]: It seems that double space is not the stable output, can you please use ` \+` between mac and ip here?

Comment (milan-zededa) [reply tests/eclient/testdata/app_dhcp.txt]: done

**Review by giggsoff — APPROVED** [review]
LGTM

**Review by rvs — COMMENTED** [review]
LGTM

**Pull request merged**